src/components/Public/Record/Importer/FileUpload.vue

```diff
@@ -91,6 +92,14 @@ export default {
 
       return this.$t('block.recordList.import.dropzoneLabel')
     },
+
+    getMaxFilesize () {
+      return this.$Settings.current.Record.Attachments.MaxSize
+    },
+
+    getAcceptedFiles () {
+      return [...this.$Settings.current.Record.Attachments.Mimetypes, ...['application/json', 'text/csv']]
```

Comment:
This will duplicate `'application/json', 'text/csv'` if they are already in the set -- `...` (spread) doesn't remove dups from array. You can use Set for this -- for the entire thing even

Reply:
If there's a duplicate there's no problem. Its just found faster with include.

Reply:
Then just use `.concat()`, this doesn't look nice imo